src/main/java/de/hysky/skyblocker/skyblock/item/ItemCooldowns.java

```diff
@@ -95,55 +107,40 @@ public static void afterBlockBreak(World world, PlayerEntity player, BlockPos po
     }
 
     private static TypedActionResult<ItemStack> onItemInteract(PlayerEntity player, World world, Hand hand) {
-        if (!SkyblockerConfigManager.get().uiAndVisuals.itemCooldown.enableItemCooldowns)
-            return TypedActionResult.pass(ItemStack.EMPTY);
-        String usedItemId = ItemUtils.getItemId(player.getMainHandStack());
-        if (usedItemId.equals(GRAPPLING_HOOK_ID) && player.fishHook != null) {
-            if (!isOnCooldown(GRAPPLING_HOOK_ID) && !isWearingBatArmor(player)) {
-                ITEM_COOLDOWNS.put(GRAPPLING_HOOK_ID, new CooldownEntry(2000));
-            }
-        }
-		if (usedItemId.equals(ROGUE_SWORD_ID)) {
-			if (!isOnCooldown(ROGUE_SWORD_ID)) {
-				ITEM_COOLDOWNS.put(ROGUE_SWORD_ID, new CooldownEntry(5000));
-			}
-		}
-		if (usedItemId.equals(INK_WAND_ID)) {
-			if (!isOnCooldown(INK_WAND_ID)) {
-				ITEM_COOLDOWNS.put(INK_WAND_ID, new CooldownEntry(30000));
-			}
+		if (!SkyblockerConfigManager.get().uiAndVisuals.itemCooldown.enableItemCooldowns)
+			return TypedActionResult.pass(ItemStack.EMPTY);
+		String usedItemId = ItemUtils.getItemId(player.getMainHandStack());
+		switch (usedItemId) {
```

Comment:
Can you sort this switch by the time while you're at it? You also have two entries for `30000`.